docs/release_docs/Pr-ci-cd_ReleaseFlow.md

```diff
@@ -0,0 +1,26 @@
+## PR-CI-CD
+ 
+ PR-CI-CD can be trigger by creating a pull request from ``` Feature/* ``` branch to merge into develop branch.
```

Comment:
The CI-CD process is mandatory, not an optional choice and our sentences should reflect the same concept.

Re-frame it as: "PR Workflow is triggered automatically whenever a PR with source branch as a `feature/*` branch is created against the target branch as `develop` branch."